packages/shared/src/errors.ts

```diff
@@ -16,5 +16,16 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+import { bip39Words as bip39WordsString } from '@/utils/bip39'
+
+const bip39Words = bip39WordsString.split(' ')
```

Comment:
Let's directly replace `' '` with `'|'` since we don't need an array anymore, like I had in the code you removed 👍 

```ts
const bip39Words = bip39WordsString.replaceAll(' ', '|')
```

Reply:
Already done yes!